jobs/gtfs-rt-parser/gtfs_rt_parser.py

```diff
@@ -193,73 +186,48 @@ def try_handle_one_feed(
         return e
 
 
-class Prefix(str, Enum):
-    al = "al"
-    tu = "tu"
-    rt = "rt"
-
-
-class RTFileType(str, Enum):
-    service_alerts = "service_alerts"
-    trip_updates = "trip_updates"
-    vehicle_positions = "vehicle_positions"
-
-
-yesterday = (date.today() - timedelta(days=1)).isoformat()
-
-
 def main(
-    filename_prefix: Prefix,
-    rt_file_substring: RTFileType,
-    iso_date: str = yesterday,
-    src_path=f"{get_bucket()}/rt/",
-    dst_path=f"{get_bucket()}/rt-processed/",
+    file_type: RTFileType,
+    glob: str = f"{get_bucket()}/rt/{yesterday}T12:*",
```

Comment:
**question:** Why do we specify a default here? I guess for testing convenience? But I would assume that we want the caller to always specify what they want to parse....

Reply:
Yeah just for testing convenience. I will remove it, it could definitely lead to unintended consequences.